backend/lib/openfisca/mapping/individu/index.ts

```diff
@@ -51,6 +51,12 @@ const individuSchema: individuGeneratorLayout = {
   },
   bourse_criteres_sociaux_distance_domicile_familial: {
     fn: function (individu, situation) {
+      if (
+        ["decedes", "sans_autorite"].indexOf(situation.parents._situation) < 0
+      ) {
+        return 0
+      }
```

Comment:
Amélioration de la lisibilité avec la destructuration de `situation.parent` (aussi utilisé l. 64) et à mon sens `!includes` est plus parlant qu'un `indexOf < 0` (l. 110 également)

```suggestion
  const { parents } = situation
  if (!['decedes', 'sans_autorite'].includes(parents?._situation)) {
    return 0
  }
```

Reply:
C'est la variable `parent` de situation qui est répétée (l. 64).

Le chaining operator permet d'éviter une erreur dans le cas où parent serait non défini ou null et de renvoyer `undefined` et par conséquence l'includes renvoie `false`